contractcourt/chain_arbitrator.go

```diff
@@ -262,6 +262,13 @@ type arbChannel struct {
 	c *ChainArbitrator
 }
 
+// ChanType returns the channel commitment type.
+//
+// NOTE: Part of the ArbChannel interface.
+func (a *arbChannel) ChanType() channeldb.ChannelType {
+	return a.channel.ChanType
```

Comment:
Could `a.channel` ever be nil?

Reply:
Shouldn't be since `arbChannel` is created via `getArbChannel` in `newActiveChannelArbitrator`.